### ISSUE TITLE
Tags for application container images

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#10618]
Prereq issues:

- #10478

As we make progress, we should update https://preview.guides.gdcc.io/en/develop/container/running/production.html

**Comment by cmbz** [comment]
2024/07/19 - 6.4 proposal request from @poikilotherm

**Comment by cmbz** [comment]
2024/11/21: Placing on hold, @pdurbin  will monitor it until @poikilotherm has time to get to it.

**Comment by pdurbin** [comment]
Yes, it's number one on our list at https://github.com/orgs/IQSS/projects/34/views/17 so we definitely won't forget it.

**Comment by pdurbin** [comment]
A new PR from @poikilotherm!

- #11477
